Linter errors

From the output of `lint_python`:

Run black --check .
would reformat tests/behavioral/test_strategy.py
would reformat tests/behavioral/test_publish_subscribe.py
would reformat tests/structural/test_proxy.py
Oh no! 💥 💔 💥
3 files would be reformatted, 59 files would be left unchanged.
Error: Process completed with exit code 1.

--- a/tests/behavioral/test_publish_subscribe.py
+++ b/tests/behavioral/test_publish_subscribe.py
@@ -27,7 +27,7 @@ class TestProvider(unittest.TestCase):
         cls.assertEqual(len(pro.subscribers[subscription]), 0)
 
     def test_publisher_shall_append_subscription_message_to_queue(cls):
-        """ msg_queue ~ Provider.notify(msg) ~ Publisher.publish(msg) """
+        """msg_queue ~ Provider.notify(msg) ~ Publisher.publish(msg)"""
         expected_msg = "expected msg"
         pro = Provider()
         pub = Publisher(pro)

--- a/tests/behavioral/test_strategy.py
+++ b/tests/behavioral/test_strategy.py
@@ -9,22 +9,14 @@ def order():
 
 
 @pytest.mark.parametrize(
-    "func, discount",
-    [
-        (ten_percent_discount, 10.0),
-        (on_sale_discount, 45.0)
-    ]
+    "func, discount", [(ten_percent_discount, 10.0), (on_sale_discount, 45.0)]
 )
 def test_discount_function_return(func, order, discount):
     assert func(order) == discount
 
 
 @pytest.mark.parametrize(
-    "func, price",
-    [
-        (ten_percent_discount, 100),
-        (on_sale_discount, 100)
-    ]
+    "func, price", [(ten_percent_discount, 100), (on_sale_discount, 100)]
 )
 def test_order_discount_strategy_validate_success(func, price):
     order = Order(price, func)
@@ -41,10 +33,7 @@ def test_order_discount_strategy_validate_error():
 
 @pytest.mark.parametrize(
     "func, price, discount",
-    [
-        (ten_percent_discount, 100, 90.0),
-        (on_sale_discount, 100, 55.0)
-    ]
+    [(ten_percent_discount, 100, 90.0), (on_sale_discount, 100, 55.0)],
 )
 def test_discount_apply_success(func, price, discount):
     order = Order(price, func)

--- a/tests/structural/test_proxy.py
+++ b/tests/structural/test_proxy.py
@@ -8,17 +8,17 @@ from patterns.structural.proxy import Proxy, client
 class ProxyTest(unittest.TestCase):
     @classmethod
     def setUpClass(cls):
-        """ Class scope setup. """
+        """Class scope setup."""
         cls.proxy = Proxy()
 
     def setUp(cls):
-        """ Function/test case scope setup. """
+        """Function/test case scope setup."""
         cls.output = StringIO()
         cls.saved_stdout = sys.stdout
         sys.stdout = cls.output
 
     def tearDown(cls):
-        """ Function/test case scope teardown. """
+        """Function/test case scope teardown."""
         cls.output.close()
         sys.stdout = cls.saved_stdout
 

